src/index.ts

```diff
@@ -0,0 +1,11 @@
+import { Parser } from "parser";
```

Comment:
index.ts, is it the entry point of the viewer or the extension ?
I'm a bit confused by your file structure